vtk/build.sh

```diff
@@ -21,26 +25,27 @@ if [ `uname` == Linux ]; then
         -DBUILD_SHARED_LIBS=ON \
         -DVTK_WRAP_PYTHON=ON \
         -DPYTHON_EXECUTABLE=${PYTHON} \
-        -DPYTHON_INCLUDE_PATH=${PREFIX}/include/python${PY_VER} \
-        -DPYTHON_LIBRARY=${PREFIX}/lib/${PY_LIB} \
+        -DPYTHON_INCLUDE_PATH=${PY_INC} \
+        -DPYTHON_LIBRARY=${PY_LIB} \
         -DVTK_INSTALL_PYTHON_MODULE_DIR=${SP_DIR} \
         -DModule_vtkRenderingMatplotlib=ON \
-        -DVTK_USE_X=ON
+        -DVTK_USE_X=ON \
+        -DVTK_PYTHON_VERSION=${PY_VER:0:1}
 fi
 
 if [ `uname` == Darwin ]; then
     CC=cc
     CXX=c++
-    PY_LIB="libpython${PY_VER}.dylib"
-    SDK_PATH="/Developer/SDKs/MacOSX10.6.sdk"
+
+    # use globs to take into account various possble suffixes: m, u, d
+    PY_LIB=`ls ${PREFIX}/lib/libpython${PY_VER}*.dylib | head -n 1`
+    PY_INC=`ls -d ${PREFIX}/include/python${PY_VER}* | head -n 1`
 
     cmake .. \
         -DCMAKE_C_COMPILER=$CC \
         -DCMAKE_CXX_COMPILER=$CXX \
         -DVTK_REQUIRED_OBJCXX_FLAGS='' \
         -DVTK_USE_CARBON=OFF \
-        -DVTK_USE_TK=OFF \
```

Comment:
I got the following warning during the build
```
CMake Warning:
  Manually-specified variables were not used by the project:

    VTK_USE_TK
```
There did seem to still be places in the VTK 7 source code that check VTK_USE_TK, though, so I will go ahead and put it back in.

It looks like the VTK_USE_CARBON flag can safely be removed as Carbon support was removed from VTK7.